pom.xml

```diff
@@ -105,7 +105,9 @@
     <commons.io.version>2.4</commons.io.version>
     <commons.collections.version>3.2.1</commons.collections.version>
     <commons.logging.version>1.1.1</commons.logging.version>
-    <shiro.version>1.2.3</shiro.version>
+    <shiro.version>1.3.2</shiro.version>
```

Comment:
Could you also update shiro version in `zeppelin-distribution/src/bin-license/LICENSE`?